code/setup.dm

```diff
@@ -25,6 +25,29 @@ var/global/disable_vents     = 0
 #define CELL_VOLUME 2500	//liters in a cell
 #define MOLES_CELLSTANDARD (ONE_ATMOSPHERE*CELL_VOLUME/(T20C*R_IDEAL_GAS_EQUATION))	//moles in a 2.5 m^3 cell at 101.325 Pa and 20 degC
 
+/////GAS FLAGS & DEFINES//////
+
+#define SPECIFIC_HEAT_AIR		20
+#define SPECIFIC_HEAT_CDO		30
+#define SPECIFIC_HEAT_PLASMA	200
+#define SPECIFIC_HEAT_NIO		40

```

Comment:
They are vars on the gases. The defines are just used for the values of the vars.